docs/introduction.rst

```diff
@@ -19,42 +31,82 @@ can be open source or proprietary (although some distribution services, like
 `Flathub <https://flathub.org/>`_, can have restrictions in this respect).
 
 The only technical requirements made by Flatpak are that applications follow a
-small number of Freedesktop standards, in order to enable desktop integration
+small number of FreeDesktop standards, in order to enable desktop integration
 (see :doc:`conventions`).
 
+Issues of current model of packaging
+------------------------------------
+
+It is important to understand the problems of the current model
+of packaging applications to understand the existence of Flatpak:
+
+- **Duplicated work packaging apps**: Many Linux distributions come with their own package
+  manager, package format and repository. This requires a lot of maintainers to package the
+  same application in various distributions, and/or the application developer to learn the
+  language of each format and then package the application in those distributions, and/or
+  ignore most distributions and package and support a couple of distributions. This makes
+  it very difficult to innovate the Linux desktop as a whole as a considerably high amount of time
+  is spent on packaging.
+- **Limited to apps that are packaged**: Not all applications are natively available
+  in every Linux distribution. If an application is not available in a specific
+  distribution, the user will have to rely on manually downloading the archive
+  of the application, extracting it and hoping the application will launch.
+- **Limited to distributions that have the apps**: The user is limited to the
+  number of distributions that have the needed applications for them
+  to properly setup their workflow. This reduces the amount of distributions
+  that can be suitable for a user.
+- **Hard to innovate in OS space**: The maintainers of the distributions have to spend a lot of
+  time packaging applications to make the distribution suitable for the end user, instead of focusing
+  on their end goals. This delays the progress of each distribution.
+
+Flatpak strives to fix the issues listed above, by conveniently enabling developers to distribute
+applications from one source and to target the entire Linux desktop.
+
 Reasons to use Flatpak
 ----------------------
 
-Flatpak has some major advantages over other approaches to distributing
-applications on Linux. First and foremost, Flatpak allows a single application
-build to be installed and run on virtually any Linux distribution. It can also
-be used in combination with `Flathub <https://flathub.org/>`_, a centralized
-service for distributing applications on all distributions. This makes it
-possible for application developers to target the entire Linux desktop market
-from one place.
+Flatpak has some major advantages over most system package managers:
+
+- **Universality**: Flatpak allows applications to be installed and run on virtually any Linux
+  distribution. This includes non-GNU distributions, systemd-free distributions,
```

Comment:
What is a non-GNU distribution?

Reply:
Distributions like Alpine Linux, that aren't GNU-based.

Reply:
Looking online it seems non-GNU is term in use so I guess it's fine. I've only heard "musl-based distros" but I suppose that term just covers libc not coreutils.